docs/en/serverless/logging/monitor-datasets.mdx

```diff
@@ -0,0 +1,51 @@
+---
+id: serverlessObservabilityMonitorDatasets
+slug: /serverless/observability/monitor-datasets
+title: Monitor log data set quality
+description: Monitor log data sets to find degraded documents.
+tags: [ 'serverless', 'observability', 'how-to' ]
+---
+
+<p><DocBadge template="technical preview" /></p>
+<p><DocBadge template="beta" /></p>
+
+The **Data Set Quality** page provides an overview of your log data sets.
+Use this information to get an idea of your overall log data set quality and find data sets that contain incorrectly parsed documents.
+Access the Datasets page from the main ((kib)) menu and go to **Stack Management** → **Data Set Quality*.
+
+{/* Update screenshot when changes are done <DocImage size="2" url="../images/logs-dataset-overview.png" alt="Screen capture of the data set overview" /> */}
+
+<DocCallOut title="Requirements">
+  Users with the `viewer` role can see the Datasets Quality summary. To see the Active Datasets and Estimated Data summaries, users need the `monitor` [index privilege](((ref))/security-privileges.html#privileges-list-indices) for the `logs-*-*` index.
+</DocCallOut>
+
+The quality of your data sets is based on the percentage of degraded documents in each data set.
+A degraded document in a data set contains the [`_ignored`](((ref))/mapping-ignored-field.html) property because one or more of its fields were ignored during indexing.
+Fields are ignored for a variety of reasons.
+For example, when the [`ignore_malformed`](((ref))/mapping-ignored-field.html.html) parameter is set to true, if a document field contains the wrong data type, the malformed field is ignored and the rest of the document is indexed.
+
+From the data set table, you'll find information for each data set such as its namespace, size, when the data set was last active, as well as the percentage of degraded docs.
```

Comment:
`Size` is not available in serverless yet, maybe we should remove the reference from here?

Reply:
Will this be available in 8.15 for stateful?

Reply:
Yes, it will be available for stateful